### PR TITLE
add jest framework

### PR DESCRIPTION
**Pull request by remotenode**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .idea
 .env
 .vscode
+node_modules
```
Comment (yahorry) [inline .gitignore:5]:
```suggestion
tests/integration/node_modules
```

Comment (yahorry) [inline .gitignore:5]: Add `tests/integration/dist` to gitignore

Comment (remotenode) [reply .gitignore:5]: I don't think this is needed, node_modules folder should not be commited in any case

Comment (remotenode) [reply .gitignore:5]: I will merge and create separate issue

**Pull request merged**